lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1371,6 +1371,32 @@ module Resources
           }
         }
       },
+      [
```

Comment:
This is already in an array, there shouldn't be another here.

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/pull/559/commits/9a5355ede6a1e2988bb7e041faf5224a1ef4acdb#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1374-R1394